lib/service/service.go

```diff
@@ -394,12 +396,12 @@ func (process *TeleportProcess) initAuthService() error {
 			srv.Addr = fmt.Sprintf("%v:%v", process.Config.AdvertiseIP.String(), port)
 		}
 		for !askedToExit {
+			sleepTime := defaults.ServerHeartbeatTTL/2 + utils.RandomDuration(defaults.ServerHeartbeatTTL/10)
+			time.Sleep(sleepTime)
```

Comment:
I don't see it (at least looking at the left side of this diff) - to me it views as simple code reshuffling. May be you did not push something?